pontoon/base/views.py

```diff
@@ -372,8 +372,9 @@ def get_translations_info(translations):
         target=t.string,
         locale=locale,
         entity=t.entity,
-        translation=t
-    ) for t in Translation.objects.filter(pk__in=changed_translation_pks).prefetch_related('entity')]
+        translation=t,
+        project=t.entity.resource.project,
```

Comment:
There's already `project` defined a few lines above, let's use it instead. It should save us a few DB hits.